### PR TITLE
Chap27

### PR DESCRIPTION
**Pull request by codinglikeagirl42**

### CODE REVIEW

**Review by gildedgardenia — CHANGES_REQUESTED** [review]
The formatting is inconsistent and CSV needs to be capitalized. Please review each page carefully before re-submitting.

File: src/chapters/tableau-part3/studio.rst [hunk 1]
```diff
@@ -1,2 +1,112 @@
...
+
+You work for TarMart corporate. TarMart has 45 department stores across the country, of various 
+sizes. Between 2010 and 2013, TarMart has tracked weekly sales of each store by department. Your 
+supervisors want some insights on which stores and departments are performing the best in weekly 
+sales.
+
+Setting Up the Studio:
+^^^^^^^^^^^^^^^^^^^^^^
+
+Download `Retail Data Analytics data set <https://www.kaggle.com/datasets/manjeetsingh/retaildataset?select=Features+data+set.csv>`__. There are 3 csv files. To download all csv files at once, click on the ``Download`` bubble next to the ``New Notebook`` bubble.  An orange arrow is pointing to the ``Download`` bubble in the image below.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part3/studio.rst:18]:
```suggestion
Download `Retail Data Analytics data set <https://www.kaggle.com/datasets/manjeetsingh/retaildataset?select=Features+data+set.csv>`__. There are 3 CSV files. To download all CSV files at once, click on the ``Download`` bubble next to the ``New Notebook`` bubble.  An orange arrow is pointing to the ``Download`` bubble in the image below.
```

File: src/chapters/tableau-part3/studio.rst [hunk 2]
```diff
@@ -1,2 +1,112 @@
...
+
+.. figure:: figures/download-multi-files.png
+   :alt: An orange arrow points to the ``Download`` bubble, highlighting the Download option of kaggle.
+
+#. Open the csv file in Tableau Public.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part3/studio.rst:23]:
```suggestion
#. Open the CSV file in Tableau Public.  
```

File: src/chapters/tableau-part3/studio.rst [hunk 3]
```diff
@@ -1,2 +1,112 @@
...
+
+Download `Retail Data Analytics data set <https://www.kaggle.com/datasets/manjeetsingh/retaildataset?select=Features+data+set.csv>`__. There are 3 csv files. To download all csv files at once, click on the ``Download`` bubble next to the ``New Notebook`` bubble.  An orange arrow is pointing to the ``Download`` bubble in the image below.
+
+.. figure:: figures/download-multi-files.png
+   :alt: An orange arrow points to the ``Download`` bubble, highlighting the Download option of kaggle.
+
+#. Open the csv file in Tableau Public.  
+#. When you open this data set, you will want to create a relationship between the 3 CSV files.
+
+   #. Each csv has a ``Store`` column.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part3/studio.rst:26]:
```suggestion
   #. Each CSV has a ``Store`` column.
```

File: src/chapters/tableau-part3/studio.rst [hunk 4]
```diff
@@ -1,2 +1,112 @@
...
+Dashboard and Finesse
+^^^^^^^^^^^^^^^^^^^^^
+
+#. Present your charts in a dashboard that summarizes your findings. 
+#. Now is a good time to update labels and colors for an easily digestible report.
+
+Finishing Touches
+^^^^^^^^^^^^^^^^^
+
+| Before you turn in your vizzes:
```
Comment (gildedgardenia) [inline src/chapters/tableau-part3/studio.rst:95]:
```suggestion
Before you turn in your vizzes:
```

File: src/chapters/tableau-part3/tableau-part3.rst [hunk 5]
```diff
@@ -1,2 +1,172 @@
...
+
+**Parameters:**
+
+`Create Parameters <https://help.tableau.com/current/pro/desktop/en-us/parameters_create.htm>`__.
+
+**Videos:**
+
+| Watch the following videos to help you learn about Tableau calculations functionality and syntax, as well as parameters.   
+
+| The calculation videos are part of a larger calculations playlist.  We suggest you explore more videos in this playlist to learn more about calculations with certain data types and in specific situations.  Keep in mind that we will not be covering LOD (Level of Detail) calculations in this class, so you may skip over those.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part3/tableau-part3.rst:57]:
```suggestion
Watch the following videos to help you learn about Tableau calculations functionality and syntax, as well as parameters.   

The calculation videos are part of a larger calculations playlist.  We suggest you explore more videos in this playlist to learn more about calculations with certain data types and in specific situations.  Keep in mind that we will not be covering LOD (Level of Detail) calculations in this class, so you may skip over those.
```

**Review by gildedgardenia — APPROVED** [review]
Approved! Got some final suggestions for you!

File: src/chapters/tableau-part3/exercises.rst [hunk 6]
```diff
@@ -1,2 +1,113 @@
...
+
+
+Table Calculation:
+^^^^^^^^^^^^^^^^^^
+
+1. How many total bike rentals occurred in the city between Jan 2011 to Dec 2012?
+
+   A. What are the running totals of bike rides between 2011-2012
+
+      #. Hint: You can use a quick calculation for this.
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/chapters/tableau-part3/exercises.rst]: Courtney Can you take a look at this.

**Pull request merged**